addons/sale_management/tests/test_sale_order.py

```diff
@@ -351,3 +352,55 @@ def test_option_price_unit_is_not_recomputed(self):
         # after changing the quantity of the product, the price unit should not be recomputed
         sale_order_with_option.order_line.product_uom_qty = 10
         self.assertEqual(sale_order_with_option.sale_order_option_ids.price_unit, 10)
+
+    def test_product_description_no_template_description(self):
+        """
+        Test case for when the product has a description, but the quotation template line does not.
+        The final sale order line should use the product's description.
+        """
+        quotation_template_no_description = self.env['sale.order.template'].create({
+            'name': 'Template with product description',
+            'sale_order_template_line_ids': [
+                Command.create({
+                    'product_id': self.product_1.id,
+                    'name': False,
+                }),
+            ],
+        })
```

Comment:
```suggestion
        quotation_template_no_description = self.empty_order_template
        quotation_template_no_description.sale_order_template_line_ids = [
            Command.create({
                'product_id': self.product_1.id,
                'name': False,
            }),
        ]
```
Try to reuse available records from the setups when possible ;).
It'll reduce test queries, reduce potential indeterminism, and sometimes slightly improve tests speed.

Reply:
Okay got it :+1: